libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts

```diff
@@ -272,4 +241,135 @@ export class ExecuteBridgeRequest {
 
     return apiUrl;
   }
+
+  private handleResponseError(error: unknown, url: string) {
```

Comment:
Exhaustive error handling added for all of:
* `got` package error codes (it does a good job of handling these with explicit classes per error type
* All Node.js error code ([ref](https://nodejs.org/dist/latest-v20.x/docs/api/errors.html#nodejs-error-codes))